libvast/src/system/node.cpp

```diff
@@ -27,192 +29,207 @@
 #include "vast/concept/printable/stream.hpp"
 #include "vast/concept/printable/to_string.hpp"
 #include "vast/concept/printable/vast/json.hpp"
+#include "vast/detail/assert.hpp"
 #include "vast/json.hpp"
 #include "vast/logger.hpp"
 
 #include "vast/system/accountant.hpp"
 #include "vast/system/consensus.hpp"
-#include "vast/system/node.hpp"
 #include "vast/system/spawn.hpp"
 
-using namespace std::string_literals;
+using std::string;
```

Comment:
Until now, we don't include `std` types into an implementation namespace. Why did you do this selectively for `std::string`?